### PR TITLE
Promote WatchBookmarks feature to GA

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref https://github.com/kubernetes/enhancements/issues/956

```release-note
Promote WatchBookmark feature to GA.
With WatchBookmark feature, clients are able to request watch events with BOOKMARK type. Clients should not assume bookmarks are returned at any specific interval, nor may they assume the server will send any BOOKMARK event during a session.
```

### CODE REVIEW

**Comment by jpbetz** [comment]
I'm in favor of this going to GA. The reduced load on etcd is certainly welcome.

**Comment by wojtek-t** [comment]
> I'm in favor of this going to GA. The reduced load on etcd is certainly welcome.

@jpbetz - I don't think the impact on etcd is actually very significant (if any). We're serving watches from apiserver anyway. So it help apiserver (and watchcache contention).

**Comment by wojtek-t** [comment]
@lavalamp @deads2k - can you please take a look?

File: staging/src/k8s.io/apiserver/pkg/features/kube_features.go [hunk 1]
```diff
@@ -161,7 +162,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	StorageVersionHash:      {Default: true, PreRelease: featuregate.Beta},
 	WinOverlay:              {Default: false, PreRelease: featuregate.Alpha},
 	WinDSR:                  {Default: false, PreRelease: featuregate.Alpha},
-	WatchBookmark:           {Default: true, PreRelease: featuregate.Beta},
+	WatchBookmark:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.21
```
Comment (lavalamp) [inline staging/src/k8s.io/apiserver/pkg/features/kube_features.go]: Remove what in 1.21? We shouldn't need to wait that long before removing the feature gate itself?

**Comment by lavalamp** [comment]
/approve

Anyone can lgtm once my question is answered :)

File: staging/src/k8s.io/apiserver/pkg/features/kube_features.go (hunk 1 above)
Comment (wojtek-t) [reply staging/src/k8s.io/apiserver/pkg/features/kube_features.go]: Removed

**Comment by wojtek-t** [comment]
@lavalamp - PTAL

**Pull request merged**